dockerfiles/examples/local/README.md

```diff
@@ -0,0 +1,20 @@
+# HPCC Local (host) storage
+
+## Directory contents
+
+### hpcc-localfile/
+
+The hpcc-localfile helm chart will provision a new Persistent Volume and a Persistent Volume Claim for each of the required HPCC storage types.
+Once installed the generated PVC names should be used when installing the HPCC helm chart.
+The values-localfile.yaml is an example of the HPCC storage settings that should be applied, after the "hpcc-localfile" helm chart is installed.
+NB: The output of installing this chart, will contain a generated example with the correct PVC names.
+
+Examples of use:
+
+*helm install localfile hpcc-localfile/*
+*helm install localfile hpcc-localfile/ --set storage.common.hostpath /home/me/hpcc-data*
```

Comment:
I tried this and it gave an error:

$ helm install local examples/local/hpcc-localfile/ --set storage.common.hostpath ~/hpcc-data
Error: expected at most two arguments, unexpected arguments: /Users/rchapman/hpcc-data

$ helm install local examples/local/hpcc-localfile/ --set storage.common.hostpath=~/hpcc-data
Error: values don't meet the specifications of the schema(s) in the following chart(s):
hpcc-localfile:
- (root): Additional property storage is not allowed

Reply:
I think it should say

--set common.hostpath=/home/me/hpcc-data

(not sure what the * is for )

Reply:
Also I suspect hostpath directory refers to a location on the minikube VM not on the local machine? If so that needs to be clarified in the instructions

Reply:
Missed that, yes it should say common.hostpath, --set common.hostpath= is what I used when testing.


* is highlight in the MD format.